services/dy-3dvis/docker-compose.devel.yml

```diff
@@ -15,21 +15,94 @@ services:
       - POSTGRES_DB=${POSTGRES_DB}
       - POSTGRES_HOST=${POSTGRES_HOST}
       - POSTGRES_PORT=${POSTGRES_PORT}
+      - SIMCORE_NODE_BASEPATH=${SIMCORE_NODE_BASEPATH}
+      - SIMCORE_HOST_NAME=${SIMCORE_HOST_NAME}
       - TEST_DATA_PATH=/home/root/test-data
-      - SERVER_PORT=${SERVER_PORT}
-      - HOST_NAME=${HOST_NAME}
+      - SERVER_PORT=8777
+      - PARAVIEW_DEBUG=0
```

Comment:
PARAVIEW_DEBUG=0 in debug?

Reply:
this is for debugging paraview webserver itself and is pretty verbose... not necessary to have that one on all the time while doing normal development.